### DISCUSSION TITLE
{{else}} failing?

### DISCUSSION THREAD

**Discussion by dcgsteve** [coddingtonbear/obsidian-web#99] [Bug Report]
If I try and use {{else}} in a template (which I believe handlebars supports in the version you are building with) then I can't save the template?

original snippet (which saves):

```
{{#if page.selectedText}}
  {{quote page.selectedText}}
{{/if}}
```

new snippet (which won't save):
```
{{#if page.selectedText}}
  {{quote page.selectedText}}
{{else}}
  {{quote page.content}}
{{/if}}
```
Any ideas why this is?

Thanks!

**Comment by coddingtonbear** [comment]
I'm afraid I'm not able to reproduce any issues with this; are you still having trouble?
